packages/components/src/dropdown/test/index.js

```diff
@@ -1,68 +1,52 @@
 /**
  * External dependencies
  */
-import { fireEvent, render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
 /**
  * Internal dependencies
  */
 import Dropdown from '../';
 
-function getButtonElement( container ) {
-	return container.querySelector( 'button' );
-}
-function getOpenCloseButton( container, selector ) {
-	return container.querySelector( selector );
-}
-
 describe( 'Dropdown', () => {
```

Comment:
I also took this opportunity to refactor the dropdown test. I thought it was a bit difficult to read and ESLint was upset thinking there were no assertions in the second test. I tried to improve the test by replacing query selectors and having matchers like `toBeVisible` over `toBeTruthy` / `toHaveLength` etc. 

I'd be curious on @tyxla's opinion here, to see if we could improve the test further and for feedback on my suggestions. 😄